pipeline/07-export.R

```diff
@@ -416,14 +569,49 @@ for (town in unique(assessment_pin_prepped$township_code)) {
   addStyle(
     wb, pin_sheet_name,
     style = style_pct,
-    rows = pin_row_range, cols = c(9, 15, 23, 25), gridExpand = TRUE
+    rows = pin_row_range, cols = c(9, 15, 23, 25, 38, 40, 42), gridExpand = TRUE
   )
   addStyle(
     wb, pin_sheet_name,
     style = style_comma,
-    rows = pin_row_range, cols = c(33, 35), gridExpand = TRUE
+    rows = pin_row_range, cols = c(35, 37), gridExpand = TRUE
+  )
+  # For some reason comp links do not get autoformatted as links, possibly
+  # due to Excel not parsing within-sheet links as hyperlinks for the purposes
+  # of styling
+  addStyle(
+    wb, pin_sheet_name,
+    style = style_link,
+    rows = pin_row_range, cols = c(39, 41), gridExpand = TRUE
+  )
+  addFilter(wb, pin_sheet_name, 6, 1:53)
+  conditionalFormatting(
+    wb, pin_sheet_name,
+    cols = c(38, 40, 42),
+    rows = pin_row_range,
+    style = c("#F8696B", "#FFFFFF", "#00B0F0"),
+    rule = c(0, 0.5, 1),
+    type = "colourScale"
+  )
+  conditionalFormatting(
+    wb, pin_sheet_name,
+    cols = 26:29,
+    rows = pin_row_range,
+    style = createStyle(bgFill = "#FF9999"),
+    rule = '$AB7!=""',
+    type = "expression"
+  )
+  # For some reason vector cols don't work with expressions, so we have
+  # to duplicate the conditionalFormatting above for the second range
+  # of columns
+  conditionalFormatting(
+    wb, pin_sheet_name,
+    cols = 30:33,
+    rows = pin_row_range,
+    style = createStyle(bgFill = "#FF9999"),
+    rule = '$AF7!=""',
+    type = "expression"
   )
-  addFilter(wb, pin_sheet_name, 6, 1:48)
```

Comment:
This `addFilter` call still exists, it's just been moved up above the `conditionalFormatting` calls.